### PR TITLE
add conda standalone

### PR DESCRIPTION
**Pull request by hadim**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
@jakirkham, it seems to be ok. Waiting for your review.

Maybe @msarahan and @forrestwaters would want to review this too?

File: recipes/conda-standalone/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CONDA_VERSION:
+  - 4.7.12
+python:
+  - 3.7
```
Comment (scopatz) [inline recipes/conda-standalone/conda_build_config.yaml]: Is it necessary to pin these here?

File: recipes/conda-standalone/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{% set name = "conda-standalone" %}
+{% set version = "4.7.12" %}
+
+package:
+  name: conda-standalone
+  version: {{ version }}
+
+source:
+  - path: ./
```
Comment (scopatz) [inline recipes/conda-standalone/meta.yaml]: Is there a tarball we can use instead?

File: recipes/conda-standalone/conda_build_config.yaml (hunk 1 above)
Comment (hadim) [reply recipes/conda-standalone/conda_build_config.yaml]: I followed https://github.com/AnacondaRecipes/conda-standalone-feedstock/tree/master/recipe

Should I remove this file?

File: recipes/conda-standalone/meta.yaml (hunk 2 above)
Comment (hadim) [reply recipes/conda-standalone/meta.yaml]: Do you want to use https://github.com/conda/conda/archive/4.7.12.tar.gz instead?

**Comment by hadim** [comment]
@scopatz LGTM

File: recipes/conda-standalone/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    folder: constructor  # [win]
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pyinstaller
+    - conda ={{ environ.get("version", 4) }}
```
Comment (scopatz) [inline recipes/conda-standalone/meta.yaml]: Why not just use {{ version }}?

**Comment by hadim** [comment]
Not sure if the OSX CI issue is related to the recipe...

**Comment by hadim** [comment]
CI is all green. Hope we're good now!

**Comment by msarahan** [comment]
@nehaljwani this is mostly all your work. Do you want to be a maintainer?

**Comment by nehaljwani** [comment]
Yes, I'd like to be a maintainer as well.

**Comment by hadim** [comment]
This PR should be good to merge.

**Comment by jakirkham** [comment]
Thanks @hadim! 😀

**Pull request merged**